substrate-node/pallets/pallet-smart-contract/src/lib.rs

```diff
@@ -437,6 +500,83 @@ pub mod pallet {
             Self::_create_name_contract(account_id, name)
         }
 
+        #[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
+        pub fn create_capacity_reservation_contract(
+            origin: OriginFor<T>,
+            farm_id: u32,
+            policy: CapacityReservationPolicy,
+            solution_provider_id: Option<u64>,
+        ) -> DispatchResultWithPostInfo {
+            let account_id = ensure_signed(origin)?;
+            Self::_create_capacity_reservation_contract(
+                account_id,
+                farm_id,
+                policy,
+                solution_provider_id,
+            )
+        }
+
+        #[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
+        pub fn update_capacity_reservation_contract(
```

Comment:
rename `capacity_reservation_contract_update` so they will appear in order from the polka UI